canvas_modules/common-canvas/src/common-properties/actions/image/image.scss

```diff
@@ -20,12 +20,9 @@
 .properties-action-image  {
 	padding: 1px 2px;
 	cursor: pointer;
-	&.right {
-		margin-left: auto;
-		float: right;
-	}
```

Comment:
I modified this CSS to work with new code for accessible controls.
For non-accessible controls, added CSS in `control-item.scss`.